rulebook.tex

```diff
@@ -1864,20 +1864,25 @@ \subsection{Tournament Phases}
 The teams will receive the true points they scored during the
 competition. The points will be accumulated in this phase and the
 teams will be ranked according to the accumulated points in descending
-order.
+order.Depending on this order, only the first 80\% (number will be always
```

Comment:
I'm not sure what a good upper bound would be. I added the other suggestions.

Reply:
Even an upper bound is not so important at this time but I agree we should clarify this point.
I suggest the following: The maximum naumber of teams which can qualify for the Playoffs is eight.
Reasoning: In case of 8 teams we will have two groups with four teams. This means that we will have 12 games. With other words: 12 hours games. In case of 10 teams we would have to manage 20 games. This seems to me impossible.
In case we have less than 7 teams qualified for the Playoffs then I agree with Tim's proposal to put them in one group. We would get 15 games in the worst case. This is hard work but still possible.